src/stacks-hierarchy/Root_PurchaseConfirmationScreen/Root_PurchaseConfirmationScreen.tsx

```diff
@@ -436,19 +436,16 @@ export const Root_PurchaseConfirmationScreen: React.FC<Props> = ({
             </View>
           </GenericSectionItem>
         </Section>
-
-        <MessageBox
-          type="info"
-          message={
-            travelDate
-              ? t(
-                  PurchaseConfirmationTexts.infoText.validInFuture(
-                    formatToLongDateTime(travelDate, language),
-                  ),
-                )
-              : t(PurchaseConfirmationTexts.infoText.validNow)
-          }
-        />
+        {travelDate && (
+          <MessageBox
+            type="info"
+            message={t(
+              PurchaseConfirmationTexts.infoText.validInFuture(
+                formatToLongDateTime(travelDate, language),
+              ),
+            )}
+          />
+        )}
```

Comment:
Should this still be here? This is just as replicated in the ticket summary and message box as the "start now" was.

Reply:
@oyvindmikkelsenatb Should we remove this one as well?

<img width="422" alt="Screenshot 2023-10-12 at 13 31 47" src="https://github.com/AtB-AS/mittatb-app/assets/1774972/bb99da7e-0cac-4f9d-b412-cf0757d64846">

Reply:
As we discussed, i vote for removing since, as you said, the start time already is shown above on the screen.

Reply:
fixed in https://github.com/AtB-AS/mittatb-app/pull/3937